library/sha1.c

```diff
@@ -290,21 +298,28 @@ void mbedtls_sha1_update( mbedtls_sha1_context *ctx, const unsigned char *input,
     if( left && ilen >= fill )
     {
         memcpy( (void *) (ctx->buffer + left), input, fill );
-        mbedtls_sha1_process( ctx, ctx->buffer );
```

Comment:
I have no idea what you are pointing out. Also I did not change L319 in either the previous or the current code, I am not sure what the issue is there either...